configure.ac

```diff
@@ -91,6 +91,24 @@ dnl Checks for libraries
 AC_CHECK_LIB(pthread, pthread_mutex_init, [],
     AC_MSG_ERROR([pthread_mutex_init() not found.  libfabric requires libpthread.]))
 
+AC_ARG_ENABLE([spinlock],
+	[AC_HELP_STRING([--enable-spinlock],
+		[Enable spinlock support (default:disabled)])])
```

Comment:
We should not ask the user to opt-in to using a higher performing synchronization mechanism.  If spinlock support is available, we should just use it.